roles/openshift_hosted/defaults/main.yml

```diff
@@ -30,3 +30,7 @@ openshift_hosted_routers:
 openshift_hosted_router_certificate: {}
 openshift_hosted_registry_cert_expire_days: 730
 openshift_hosted_router_create_certificate: False
+
+os_firewall_allow:
+- service: Docker Registry Port
+  port: 5000/tcp
```

Comment:
Yes it does, we tried top push Images to the registry without any success until we have opened the port configured in the registry deployment.